tests/baselines/reference/isolatedDeclarations.errors.txt

```diff
@@ -0,0 +1,8 @@
+error TS5053: Option 'outFile' cannot be specified with option 'isolatedDeclarations'.
+
+
+!!! error TS5053: Option 'outFile' cannot be specified with option 'isolatedDeclarations'.

```

Comment:
Without `outFile` set, you can still have global script input files - just don't write any exports and ensure `module` isn't `nodenext` or `node16`. Checking for this option, instead of issuing an error in `checkSourceFileWorker` on global non-ambient files, still feels like checking the wrong thing.

Reply:
Removed the restriction